src/test/java/com/mkopec/ticketbookingapp/service/SeatValidationServiceTest.java

```diff
@@ -0,0 +1,147 @@
+package com.mkopec.ticketbookingapp.service;
+
+import com.mkopec.ticketbookingapp.domain.RoomSeat;
+import com.mkopec.ticketbookingapp.domain.Seat;
+import com.mkopec.ticketbookingapp.exception.ArgumentNotValidException;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.runners.Enclosed;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.runners.Parameterized.Parameters;
+
+@RunWith(Enclosed.class)
+public class SeatValidationServiceTest {
```

Comment:
jeżeli miałbyś napisać jeden test, to jaki byłby to test? Test tego, że osoba wyświetla filmy, wybiera sens, dokonuje rezerwacji i system tą rezerwację przyjmuje; czy testy walidacji zajętych miejsc?

Reply:
Przyznaję, że brakuje tutaj testów use casów (same w sobie zawierałyby kilka testów rezerwacji miejsc). A test napisany z powodu dużej ilości kombinacji ustawień siedzeń, ciężkie do sprawdzenia w sposób ręczny.
Więc jeśli miałbym wybierać to test use case - sprawdza więcej elementów.